### DISCUSSION TITLE
FB/Twit/Insta icons not appearing when pasting code

### DISCUSSION THREAD

**Discussion by JamesUccello** [twbs/bootstrap#38820] [Q&A]
First off, Im a newbie. When I copy the code for a footer the icons dont appear. I have tried to inspect, then highlight the element, copy, then paste however these dont show. Here is screenshots of the steps I took. Everything else work just not these icons.

![footer](https://github.com/twbs/bootstrap/assets/130889337/88560a51-d77d-412e-9845-78bf22c0e695)
![footer2](https://github.com/twbs/bootstrap/assets/130889337/d71ad0e2-5bb8-4dd7-bf8e-0c377e642706)
![footer3](https://github.com/twbs/bootstrap/assets/130889337/4437cd73-77c1-4570-a242-6492ac41ca73)
![footer4](https://github.com/twbs/bootstrap/assets/130889337/e9d99673-5982-487b-ba7c-92755249c95a)

**Comment by julien-deramond** [comment]
Hey @cpdigicomm 

Here's a StackBlitz corresponding to the Bootstrap Footers example: https://stackblitz.com/edit/s3vai2?file=index.html.
Some classes might be a little different because it's based on a previous version of Bootstrap, but the icons are well displayed here.
Hope it can help!
